cmd/execute.go

```diff
@@ -140,15 +140,17 @@ var executeCmd = &cobra.Command{
 				errMsg = fmt.Sprint("SummaryError: ", execErr)
 				exitCode = ExitCodeSummaryTypeError
 			case *multierror.Error:
-				mErr := cause.(*multierror.Error)
-				errCount := len(mErr.Errors)
-				if mErr != nil && errCount > 0 {
-					errMsg = fmt.Sprintf("%d errors occurred:\nFirst error: %s", errCount, mErr.Errors[0].Error())
+				mErr := c
```

Comment:
pointlesss allocation, just use variable already allocated, (and maybe call it somehting with err instead of `c`).